contents/blog/product-engineer-tools.md

```diff
@@ -0,0 +1,90 @@
+
+To talk with users, product engineers use tools like Zoom, Google Meet, Slack (especially shared channels), [Superhuman](https://superhuman.com/), [Cron](https://cron.com/) (calendar), Calendly, and [SavvyCal](https://savvycal.com/). They also take advantage of automation by using tools like [customer.io](https://customer.io/), aha.io, [n8n.io](https://n8n.io/), and Zapier.
+
+Unlike product managers, product engineers don’t use survey tools. The data received from surveys is too indirect to be useful. It might be valuable for product and marketing teams to get a general idea of users, but product engineers focus on direct feedback on what they are building.
+
+## Get data on how users are using the product
+
+Product engineers supplement talking to users with product analytics and usage data. This data enables them to measure the impact of their work and discover new areas for improvement. It helps them set goals, measure key results, and prioritize future work.
+
+For example, a product engineer sets an OKR to increase feature usage by paid users. They need data to both measure the usage of the feature, and areas surrounding the feature. With this, they can analyze conversion to using the feature, feature retention, correlated behavior to feature usage, and more.
+
+[PostHog](/product#top-features) has a suite of product analytics tools that are perfect for this. Product engineers (like [Y Combinator’s](/customers/ycombinator)) use PostHog to capture, organize, analyze, and visualize product usage data. PostHog also has session recordings for going into the details of an individual’s experience with the product.
+
+On top of PostHog, product engineers often use customer data platform or ETL tools like [Segment](/docs/integrate/third-party/segment), [Hightouch](/blog/hightouch-posthog-reverse-etl-integration), [Airbyte](/docs/apps/airbyte-export), Integrate.io, and Fivetran to move and combine data between sources and storage locations. 
+
+## Using knowledge to prioritize properly and stay on track
+
+Once product engineers talk with users and analyze usage data, they should be starting to form ideas on what is impactful to build. All this research and analysis is wasted if it does not lead to shipping better products.
+
+Product engineers use note-taking and task management tools to help them with this. Many of these tools can weigh teams down and introduce unnecessary work that gets in the way of shipping products that help users (**cough** Jira). The tools for product engineers ideally don’t do this.
+
+An example specifically built for product engineers is [Linear](https://linear.app/). Linear focuses on helping teams build great products. It does this by having engineer-focused design and functionality, while still being a product tool. In practice, this means being keyboard first, streamlined for technical use cases, and fast loading and usage speeds.
+
+![Linear](../images/blog/product-engineer-tools/linear.png)
```

Comment:
Feels a bit weird this is the only tool we have a screenshot for. Could we maybe have one screenshot per section with a caption for each?

Also, any chance we could get a good screenshot of the product rather than the website?

Reply:
Added more images throughout. I chose that Linear image because it explained why Linear was good for product engineers, and a screenshot of the product might not. Can change it is you disagree.